web/src/main/java/no/nav/foreldrepenger/web/app/tjenester/fpoversikt/SvpDtoTjeneste.java

```diff
@@ -160,17 +166,16 @@ private Set<SvpSak.Vedtak.ArbeidsforholdUttak> finnArbeidsforhold(BehandlingVedt
                             SvpSak.Vedtak.ArbeidsforholdUttak.SvpPeriode.ResultatÅrsak.AVSLAG_INNGANGSVILKÅR;
                     };
 
-                    var arbeidstidprosent =  matchendeTilretteleggingFOM
-                        .filter(mt -> TilretteleggingType.DELVIS_TILRETTELEGGING.equals(mt.getType()))
+                    var arbeidstidprosent = matchendeTilretteleggingFOM.filter(mt -> TilretteleggingType.DELVIS_TILRETTELEGGING.equals(mt.getType()))
                         .map(TilretteleggingFOM::getStillingsprosent)
                         .orElse(null);
                     return new SvpSak.Vedtak.ArbeidsforholdUttak.SvpPeriode(p.getFom(), p.getTom(), tilretteleggingType, arbeidstidprosent,
                         p.getUtbetalingsgrad().decimalValue(), resultatÅrsak);
                 }).filter(Objects::nonNull).collect(Collectors.toSet());
                 //utlede oppholdsperioder
                 var oppholdsperioder = matchendeTilrettelegging.map(mt -> finnAlleOppholdsperioderFraTlr(mt, behandling)).orElse(Set.of());
-
-                return new SvpSak.Vedtak.ArbeidsforholdUttak(new SvpSak.Aktivitet(type, arbeidsgiver, arbeidsforholdId),
+                var arbeidsgiverNavn = Optional.ofNullable(arbeidsgiverTjeneste.hent(ua.getArbeidsgiver())).map(ArbeidsgiverOpplysninger::getNavn).orElse(null);
```

Comment:
Vi du håndtere not found i frontend - eller bruke en "Finner ikke navn" her?

Reply:
Det jeg har gjort per nå er at jeg fallbacker til å vise orgnr hvis ikke navnet er gitt. Enten fordi vi ikke finner den her, eller for gamle umigrerte saker i fpoversikt som ikke har navnet